.storybook/manager.js

```diff
@@ -0,0 +1,8 @@
+import { addons } from '@storybook/addons'
+import theme from './theme'
+
+addons.setConfig({
```

Comment:
The `manager.js` file configures the actual Storybook manager appearance (i.e. everything _except_ for how the stories are rendered).